JS/Lesson30/onder/my-app/src/components/Ex7/İndex.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './index.css';
+
+export const Ex7 = () => {
+  function parentHandleClick() {
+    console.log("Parent element clicked");
+  }
+
+  function childHandleClick(event) {
+    event.stopPropagation();
+    console.log("Child element clicked");
+  }
+
+  return (
+    <>
```

Comment:
You only need to use Fragment component (<></>) in React if you have more than one element on the same level that you need to return. Here you only have <div> so no need to wrap it in <></>.